admin/server/magic_tokens.go

```diff
@@ -49,13 +47,19 @@ func (s *Server) IssueMagicAuthToken(ctx context.Context, req *adminv1.IssueMagi
 		return nil, status.Error(codes.PermissionDenied, "not allowed to create a magic auth token")
 	}
 
+	resources := make([]database.ResourceName, len(req.Resources))
```

Comment:
It seems like it's missing backwards compatibiltiy for `req.ResourceType` and `req.ResourceName`? (Can be fixed by adding them to `req.Resources` if they are not empty)